packages/locale/src/format/cldr/CldrCache.js

```diff
@@ -0,0 +1,54 @@
+/*
+* @copyright (c) 2020, Philipp Thuerwaechter & Pattrick Hueper
+* @license BSD-3-Clause (see LICENSE.md in the root directory of this source tree)
+*/
+
+import cldrData from 'cldr-data';
+import Cldr from 'cldrjs';
+
+const cldrDataLoaded = new Set();
```

Comment:
This 'cache' seems harmless to me, it's just a set of strings passed to `cldrData()` so I imagine the size should be pretty trivial.